build_tag.sh

```diff
@@ -6,14 +6,18 @@ set -exvo pipefail -o nounset
 # - sync secrets from app-interface:
 #   -> secrets: /resources/jenkins/global/secrets.yaml
 #   -> for job `gh-build-tag`: /resources/jenkins/global/templates.yaml
-IMAGE_TEST=addon-metadata-operator
-docker build -t ${IMAGE_TEST} -f Dockerfile.ci .
+#   -> doing cross-comiple builds
 docker_run_args=(
     --rm
-    # goreleaser version
-    -e "VERSION=v0.184.0"
+    --privileged
+    -e CGO_ENABLED=1
```

Comment:
I would not set CGO_ENABLED globally. Might lead to weird result when goreleraser is being installed or pulling dependencies. It is set by goreleaser in `go build ...` command and that's enough.

Reply:
unless this is specific to `goreleaser/goreleaser-cross` and part of their documentation?

Reply:
it is using docker image to build binaries and not using any of our commands to do it. so, yes this -e is required to inform goreleaser-cross